src/components/MonitorList.vue

```diff
@@ -129,6 +156,15 @@ export default {
 
             return result;
         },
+
+        isDarkTheme() {
```

Comment:
Not particularly happy about having this here. Wanted to import the theme mixin, but it contains side-effects.